README.md

```diff
@@ -22,7 +22,7 @@ To see sample implementation code of the PowerDMS File Picker in further detail,
     <script src="https://filepicker.powerdms.com/initializer/powerDmsFilePicker.js"></script>
     ```
 
-2. Add a button to the page that will call `window.initializePowerDmsFilePicker(config)`.
+2. Add a button to the page that will call `window.PowerDms.initializePowerDmsFilePicker(config)`.

```

Comment:
Add something describing the fact that there is a top level namespace that will house all PowerDms api calls.